packages/infrastructure/webpack/src/get-react-config.js

```diff
@@ -1,13 +1,16 @@
 const getNodeConfig = require("./get-node-config");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const path = require("path");
+
 
 module.exports =
-  ({ miniCssExtractPluginLoader = MiniCssExtractPlugin.loader } = {}) =>
+  ({ miniCssExtractPluginLoader = MiniCssExtractPlugin.loader, dependencies = { resolvePath: path.resolve, scriptDirectory: __dirname } } = {}) =>
   ({ entrypointFilePath, outputDirectory }) => {
     const nodeConfig = getNodeConfig({
       entrypointFilePath,
       outputDirectory,
     });
+    const sassCommonVars = dependencies.resolvePath(dependencies.scriptDirectory, "..", "..", "..", "core", "src", "renderer", "components/vars.scss");
```

Comment:
Does `"components/vars.scss"` need to be `"components"` and `"vars.scss"`?

Reply:
Should work like this. It has been copied from open-lens package where I believe it has been tested

https://github.com/lensapp/lens/blob/master/packages/open-lens/webpack/vars.ts#L19

Reply:
this has been removed